brakeman.gemspec

```diff
@@ -15,6 +15,7 @@ Gem::Specification.new do |s|
   s.cert_chain  = ['brakeman-public_cert.pem']
   s.signing_key = gem_priv_key if File.exist? gem_priv_key and $0 =~ /gem\z/
   s.add_development_dependency "test-unit"
+  s.add_development_dependency "byebug"
```

Comment:
I'm obviously fine with ripping this out, but I always find myself adding this line temporarily when working on brakeman. I'm not at all committed to byebug specifically, but bundling in a reasonable debugger is nice for dev.